### PR TITLE
Adds a new traitor item: The Syndicate Ventilation Specialist Kit

### PR DESCRIPTION
**Pull request by TeomanTheGreat**
## About The Pull Request
Adds the Syndicate Ventilation Specialist Kit which contains an implant that allows the user to ventcrawl even with clothes on, a syndicate spacesuit (not a hardsuit), and a combat knife. 

I am not 100% sure on the price btw so it would be nice if people give me some suggestions for it. 
Edit: Just made it 20tc since ventcrawl is rather strong

## Why It's Good For The Game
New traitor items good

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

**The kit in the uplink:**
image
**The box:**
image
**Inside the box + the autosurgeon for the implant**
image
**Venting:**
image
After removing the implant via the VV editor i can no longer ventcrawl. 
</details>

## Changelog
:cl: Phil Smith
add: Adds the Syndicate Ventilation Specialist Kit which contains an implant that allows the user to ventcrawl even with clothes on, a syndicate spacesuit (not a hardsuit), and a combat knife. 
/:cl:

### CODE REVIEW

**Comment by FortSumter** [comment]
Allow them to wear anything that doesnt fit in the exosuit slot, make it 18TC, and I'd think you'd be good. Ventcrawl can be very broken, but with only  2TC to work with and the inability to wear armor I think it'd be good.

**Comment by TeomanTheGreat** [comment]
> Allow them to wear anything that doesnt fit in the exosuit slot, make it 18TC, and I'd think you'd be good. Ventcrawl can be very broken, but with only 2TC to work with and the inability to wear armor I think it'd be good.

im not exactly sure how i would go about coding that since i am not all that experienced in DM. also the red spacesuit goes on the exosuit slot if they want to use it during their venting.

**Comment by Isy232** [comment]
R-RED VENTED... SUSSY IMPOSTER AMOGUS

**Comment by FortSumter** [comment]
I'm not a beyond coder and I have minimal qualifications as the only coding I've done was in college. The monkey ventcrawl mechanics allow them only to vent if they have no items on them. Perhaps that code could be reworked for this purpose?

**Comment by FortSumter** [comment]
> the red spacesuit goes on the exosuit slot if they want to use it during their venting.

I'm guessing that this is an among us refference like the last guy thought. Realistically like I said it is VERY OP to be able to vent crawl at all, let alone have enough TC for a space suit as well or any kind of armor. I just think you'd have an easier time getting this approved and an easier time coding it if you just excluded all items that go in the exo slot rather than excluding certain categories.

**Comment by TeomanTheGreat** [comment]
> > the red spacesuit goes on the exosuit slot if they want to use it during their venting.
> 
> I'm guessing that this is an among us refference like the last guy thought. Realistically like I said it is VERY OP to be able to vent crawl at all, let alone have enough TC for a space suit as well or any kind of armor. I just think you'd have an easier time getting this approved and an easier time coding it if you just excluded all items that go in the exo slot rather than excluding certain categories.

asking in the coding channel on the discord now to figure out how to implement this. ill probably replace the spacesuit with a red jumpsuit if i figure this out.

**Comment by bluezorua** [comment]
make the implant cause both your arms to disappear, maybe even your head

**Comment by Raven-Industries** [comment]
Perhaps call it the "HVAC technician kit".

File: code/modules/surgery/organs/augments_chest.dm [hunk 1]
```diff
@@ -207,3 +207,20 @@
...
+	implant_color = "#fffb00"
+	syndicate_implant = TRUE
+
+/obj/item/organ/cyberimp/chest/ventcrawler/Insert()
+	. = ..()
+	owner.ventcrawler = VENTCRAWLER_ALWAYS
+
+/obj/item/organ/cyberimp/chest/ventcrawler/Remove(mob/living/carbon/M, special = FALSE)
+	. = ..()
+	M.ventcrawler = VENTCRAWLER_NONE
```
Comment (DrDuckedGoose) [inline code/modules/surgery/organs/augments_chest.dm]: You might want to consider this, just so monkeys don't get vent privileges removed.

```
/obj/item/organ/cyberimp/chest/ventcrawler/Remove(mob/living/carbon/M, special = FALSE)
	. = ..()
	M.ventcrawler = initial(M.ventcrawler)
```

**Comment by Fronsis** [comment]
I think someone tried to make a pr like this but got shutdown because '' too meme-y'' (while we still have other video game references present lol)

Eh idk i like it, it gives you a different gimmick to try and expands possibilities (Imagine venting into perma to give a prisoner tools to escape) i do agree that it needs to limit what you can carry while inside the vent (For example only the two-three slots of the suit) to make it fair and not too broken

**Comment by PowerfulBacon** [comment]
Vent crawl is probably not worth 20tc when compared to other traitor items. I can teleport into the armoury for significantly cheaper

**Comment by FortSumter** [comment]
Bacon I'm not seeing where anyone suggested 20TC. My idea is between 15 and 18TC. I think ventcrawl is very powerful considering you could pair this with a shotgun and erase most heads of staff in their offices then disappear into the vents with little to counter it. The very least we can do is make sure they can't combo it with other traitor items.

**Comment by PowerfulBacon** [comment]
> Bacon I'm not seeing where anyone suggested 20TC.

The thing is 20 TC, I'm not looking at suggestions just what is actually being implemented.

I think like the last among us meme, its just too memey. Perhaps if it was a syndicate kit item, it could be more acceptable. Going to put a key on this for now.

**Comment by ike709** [comment]
🔑 sus

**Comment by DrDuckedGoose** [comment]
Fucking travesty

**Comment by TeomanTheGreat** [comment]
too sus 😔

**Comment by itsmeow** [comment]
we need amomg us please revive

**Pull request closed without merging**